.github/workflows/Helix-PR-CI.yml

```diff
@@ -0,0 +1,46 @@
+# Run the mininum required tests for pull requests.
```

Comment:
Will this block PR merges upon test failure? If so, then we should reconsider adding this because it may not be reasonable to assume all tests will pass consistently.

Reply:
I don't think it will block anything. Since I can still merge code to my forked repo. I think this just provides additional support.
On the other hand, if it is not the case now, then we shall delay this PR. But we will need this eventually.